landscape.yml

```diff
@@ -2280,12 +2280,12 @@ landscape:
             twitter: 'https://twitter.com/coreos'
             crunchbase: 'https://www.crunchbase.com/organization/red-hat'
           - item:
-            name: Telekube
+            name: Gravity
             description: >-
-              Telekube combines a production hardened deployment of Kubernetes with Teleport, our multi-region SSH server, so you can effectively manage
-              multiple deployments of Kubernetes applications across various regions, data centers and cloud providers.
+              Bring Kubernetes Down to Earth. Gravity is an upstream Kubernetes packaging solution that takes the drama out of on-premise deployments.
```

Comment:
Please don't include a description and just let the landscape app pull the description from your GitHub repo.

Reply:
Ok, removed description.